qiskit_metal/renderers/renderer_elmer/elmer_configs.py

```diff
@@ -0,0 +1,64 @@
+materials = dict(
```

Comment:
There isn't a single comment in this file.  If these values are coming from somewhere else, at least provide a link. So future contributors know where to look. Otherwise, please explain the purpose of each dict?

Reply:
Great question @priti-ashvin-shah-ibm. 
I talked to @AbeerVaishnav13 and he explained that the reason he structured it this way was because not all of the options under `elmer_configs.py` need to be accessible to the user. the `default_options` and `default_setup` to start the renderer are the ones which the user should be able to change, but for example, the type of equation to be solved is something that we still need to provide to ElmerFEM, but the user shouldn't modify for a given analysis.
He also mentioned that these dictionaries could be moved into either `elmer_renderer.py` or `elmer_runner.py`, but as we add more solvers and materials to this database, then the files with the relevant code will grow longer. 

Please let us know if you think this should be structured differently and we can definitely change it.

Reply:
Thanks for clarification.  I see no reason to change this now.